src/Discord.Net/Entities/Invites/IInvite.cs

```diff
@@ -0,0 +1,22 @@
+// todo: docs
+using System.Threading.Tasks;
+
+namespace Discord
+{
+    public interface IInvite : IEntity<string>, IDeletable
+    {
+        ValueTask<IChannel> GetChannelAsync(StateBehavior stateBehavior, RequestOptions? options = default);
+
+        string Code => Id; // review: is this worth having
```

Comment:
Seems needlessly verbose.

Reply:
That suggests to me that `IInvite` should not extend `IEntity<T>`.